### PR TITLE
mk->os-50: popup component

### PR DESCRIPTION
**Pull request by mkunak**

### CODE REVIEW

File: src/Components/Alert/index.tsx [hunk 1]
```diff
@@ -28,6 +30,7 @@ type Props = {
 }
 
 const defaultProps: Props = {
+  theme: Theme.LIGHT,
```
Comment (Tima0203) [inline src/Components/Alert/index.tsx:33]: forgot to mention, make default DARK

File: src/Components/Button/index.tsx [hunk 2]
```diff
@@ -1,33 +1,40 @@
...
-  className: '',
   style: { color: '', borderColor: '' },
   onClick: () => { },
 }
 
 const ButtonComponent: FC<Props> = (props: Props) => {
-  const { text, className, ...rest } = generateRenderProps(defaultProps, props)
+  const { text, theme, className, ...rest } = generateRenderProps(defaultProps, props)
+
+  console.log('>>> colors:', generateThemeColors(theme))
```
Comment (Tima0203) [inline src/Components/Button/index.tsx:31]: remove console.log

File: src/Components/Button/index.tsx [hunk 3]
```diff
@@ -1,33 +1,40 @@
...
+// import styles from './Button.module.scss'
+import './Button.scss'
 
 interface Props {
+  theme: Theme
   text?: string
   variant?: string
   className?: string
-  style?: { color: string, borderColor: string }
+  style?: CSSProperties
```
Comment (Tima0203) [inline src/Components/Button/index.tsx:15]: add "colors" prop which is 
```
type ColorScheme = {}

export type Colors = {
  [theme in Theme]?: ColorScheme
}
```
You did it right in types so we should pass {DARK: colorSchemeDark, LIGHT: colorSchemeLight}
so then for example backgroundColor = props.colors[theme]. backgroundColor
It allows us to change theme from parent component to switch colors.

**Review by Tima0203 — CHANGES_REQUESTED** [review]

File: src/Components/Button/index.tsx (hunk 2 above)
Comment (mkunak) [reply src/Components/Button/index.tsx]: done

File: src/Components/Alert/index.tsx (hunk 1 above)
Comment (mkunak) [reply src/Components/Alert/index.tsx]: done

**Review by Tima0203 — APPROVED** [review]

**Pull request merged**